ws/raflow.go

```diff
@@ -416,9 +417,18 @@ type RARentableFeesDataRequest struct {
 // RARentableFeesDataListResponse for listing down all RARentableFeesData
 // in the grid
 type RARentableFeesDataListResponse struct {
-	Status  string               `json:"status"`
-	Total   int64                `json:"total"`
-	Records []RARentableFeesData `json:"records"`
+	RID          int64                `json:"RID"`
```

Comment:
@abataub you can remove this struct, if it's no longer needed!

Reply:
ok.